substrate/client/consensus/manual-seal/src/consensus/aura.rs

```diff
@@ -1,91 +1,399 @@
 // This file is part of Substrate.
 
+
+
 // Copyright (C) Parity Technologies (UK) Ltd.
+
+
 // SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
 
+
 // This program is free software: you can redistribute it and/or modify
+
 // it under the terms of the GNU General Public License as published by
+
 // the Free Software Foundation, either version 3 of the License, or
+
+
 // (at your option) any later version.
 
+
+
+
+
 // This program is distributed in the hope that it will be useful,
+
+
+
+
+
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
+
+
+
+
+
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+
+
+
+
+
+
+
+
 // GNU General Public License for more details.
 
+
+
+
+
+
+
+
+
+
+
 // You should have received a copy of the GNU General Public License
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Aura consensus data provider, This allows manual seal author blocks that are valid for
+
+
+
+
+
+
+
+
+
+
+
+
+
+//! Aura consensus data provider, This allows manual seal author blocks that are va
+
+
+
+
+
+
+
+
+lid for
+
+
+
+
+
+
+
+
+
+
+
+
 //! runtimes that expect the aura-specific digests.
 
+
```

Comment:
something exploded with the newlines

Reply:
Whoops, was using the online GH editor, somehow got messed up.